website/src/pages/Blog/styles.ts

```diff
@@ -8,58 +8,75 @@ const useStyles = makeStyles((theme: Theme) => ({
     backgroundRepeat: "no-repeat",
     backgroundPosition: "center top",
     backgroundSize: "cover",
-    padding: theme.spacing(12, 5, 0),
+    padding: theme.spacing(16, 5, 0),
   },
-  tabLayout: {
-    orientation: "vertical",
+  tabRoot: {
+    justifyContent: "center",
+    '& .Mui-selected': {
+      color: theme.palette.text.hint,
+      padding: theme.spacing(2, 1),
+    }
+  },
+  scroller: {
+    flexGrow: 0,
+    background: theme.palette.background.paper,
+    boxShadow: '2px 0px 33px 5px rgba(70, 68, 151, 0.04)',
+    borderRadius: '12px 12px 12px 0px'
+  },
+  breadCrumbs: {
+    '& .MuiBreadcrumbs-ol':{
+      justifyContent: 'center',
+      color: theme.palette.info.light,
+      fontSize: '14px',
+      fontWeight: 400
+    }
   },
   authorWrapper: {
     display: "flex",
     justifyContent: "center",
     alignItems: "center",
+    margin: theme.spacing(3, 0, 2.5),
   },
   large: {
-    width: theme.spacing(9),
-    height: theme.spacing(9),
-    marginRight: theme.spacing(4),
+    width: '60px',
+    height: '60px',
+    marginRight: theme.spacing(1.8),
   },
   tabs: {
-    maxWidth: "auto",
     margin: `${theme.spacing(1)}px auto`,
-    padding: theme.spacing(0.3),
-    backgroundColor: theme.palette.common.white,
+    background: 'transparent',
     flexGrow: 1,
     fontWeight: 700,
-    boxShadow: "2px 0px 33px 5px rgba(70, 68, 151, 0.04)",
+    boxShadow: 'none',
+    position: 'relative',
+    top: '25px'
   },
   mainText: {
     display: "flex",
     flexWrap: "wrap",
     textAlign: "center",
-    margin: theme.spacing(8, 50),
+    maxWidth: '670px',
+    margin: 'auto',
     justifyContent: "center",
     alignItems: "center",
-    [theme.breakpoints.down("md")]: {
-      margin: theme.spacing(8, 30),
-    },
+    fontSize: '42px',
     [theme.breakpoints.down("sm")]: {
-      margin: theme.spacing(8, 10),
-    },
-    [theme.breakpoints.down("sm")]: {
-      margin: theme.spacing(4, 0),
       fontSize: "24px",
     },
   },
   authorText: {
     display: "flex",
     flexWrap: "wrap",
     textAlign: "center",
-    margin: theme.spacing(3, 0),
+    margin: theme.spacing(0),
     justifyContent: "center",
     alignItems: "center",
+    fontSize: '42px'
```

Comment:
shouldn't we add a responsive fontsize for this one

Reply:
We had added separate font size for mobile devices as well. Now converted both font-size units in `rem`